lib/reporter.js

```diff
@@ -21,7 +21,7 @@ module.exports = class Reporter {
     for (const [i, val] of dups.entries()) {
       str += title2(`\nDuplicate ${i + 1}\n\n`);
       for (const { id, node } of val) {
-        str += subtitle(`${id} → ${this.getLocation(node)}\n`);
+        str += subtitle(`${id}:${this.getLocation(node)}\n`);
```

Comment:
but it wont point when we click the url from cli.. this makes it handle to navigate to the exact line.